tests/test_async_qdrant_client.py

```
@@ -20,7 +21,9 @@
 @pytest.mark.asyncio
 @pytest.mark.parametrize("prefer_grpc", [True, False])
 async def test_async_qdrant_client(prefer_grpc):
-    client = AsyncQdrantClient(prefer_grpc=prefer_grpc, timeout=15)
+    major, minor, patch, dev = read_version()
```

Ruff (0.15.7)

[warning] Unpacked variable `major` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)